### PR TITLE
solution and testing for task 3 GET /api

### PR DESCRIPTION
**Pull request by DeeSzubert**
solution and testing for task 3 GET /api

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 const express = require("express");
 const app = express();
 const { getAlltopics } = require("./controllers/topics-controllers");
+const endpoints = require("./endpoints.json");
+
+app.get("/api", (request, response, next) => {
+  response.status(200).send({ endpoints });
```
Comment (nc-rob-anderson) [inline app.js:7]: Well done for sending data back on a semantic key :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -27,6 +28,17 @@ describe("/api/topics", () => {
...
+describe("/api", () => {
+  test("GET200: Responds with all the available endpoints of the api", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual({ endpoints });
+      });
+  });
+});
```
Comment (nc-rob-anderson) [inline __tests__/app.test.js:40]: Good test

**Review by nc-rob-anderson — COMMENTED** [review]
Great 🚀

**Comment by DeeSzubert** [comment]
Thank you Rob

⁣Get BlueMail for Android ​

**Pull request merged**